app/uk/gov/hmrc/gform/fileupload/FileUploadController.scala

```diff
@@ -251,14 +255,34 @@ class FileUploadController(
 
             for {
               _ <- fileUploadService.deleteFile(cacheWithFileRemoved.form.envelopeId, fileToDelete)
-              res <-
-                fastForwardService
-                  .redirectFastForward[SectionSelectorType.Normal](
-                    cacheWithFileRemoved,
+              _ <- gformConnector
+                     .updateUserData(
+                       FormIdData.fromForm(cache.form, maybeAccessCode),
+                       UserData(
+                         cache.form.formData,
+                         cache.form.status,
+                         cache.form.visitsIndex,
+                         cache.form.thirdPartyData,
+                         cache.form.componentIdToFileId
```

Comment:
Why is this using `cache.form` and not `cacheWithFileRemoved` ?

Reply:
Not sure how it got there, probably messed up during commit. Fixed now